config/services.yaml

```diff
@@ -37,6 +37,9 @@ services:
                 - '@JoliCode\SecretSanta\Application\SlackApplication'
                 - '@JoliCode\SecretSanta\Application\DiscordApplication'
 
+    Jolicode\SecretSanta\Statistics:
+            public: true
```

Comment:
Il y a un niveau d'indentation de trop il me semble

Reply:
1. Are you sure it needs to be public ?
2. Are you sure you need to declare it ? (because there is autodiscovery in this project)